x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/detail_panel/detail_panel.js

```diff
@@ -34,25 +40,63 @@ export class DetailPanelUi extends Component {
     super(props);
   }
 
-  renderTabs() {
+  renderCluster() {
     const { cluster } = this.props;
+    const { connected, seeds, num_nodes_connected: connectedNodesCount } = cluster;
 
-    if (!cluster) {
-      return;
-    }
-
-    return (
-      <div>Remote cluster details</div>
-    );
-  }
-
-  renderCluster() {
-    // const { cluster, intl } = this.props;
+    const renderedSeeds = seeds.map(seed => <EuiText>{seed}</EuiText>);
```

Comment:
I think the `EuiText` needs to have a unique `key` prop:

![image](https://user-images.githubusercontent.com/1965714/48576971-cc189100-e8ca-11e8-8483-647d8d383edc.png)

Reply:
Good catch!